### PR TITLE
Card Created- Krushna

### PR DESCRIPTION
**Pull request by kru2710shna**

### CODE REVIEW

**Review by ibrahimdevop21 — CHANGES_REQUESTED** [review]
there's a redundancy issue, you have the portfolio link both in the contact section and the resources section ,  try to add any of your favorite learning resources

**Comment by kru2710shna** [comment]
Changes are made. Can you check please

**Review by ibrahimdevop21 — CHANGES_REQUESTED** [review]
the twitter link  and handle is wrong
the github handle is wrong ( just exclude Github: from the handle
missing 2 more resources.
optional: missing ending comment and a line break before and after the card

**Comment by kru2710shna** [comment]
I don't have twitter account!

File: index.html [hunk 1]
```diff
@@ -1147,7 +1147,27 @@ <h2>A project for first-time contributions</h2>
...
-
+        <!-- ========= Card 13 =========  -->
+        <!-- ========= Krushna =========  -->
+        <div class="card">
+          <p class="name">Krushna</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://aesthetic-peony-a18438.netlify.app" target="_blank">Portfolio</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/kru2710shna" target="_blank">kru2710shna</a>
```
Comment (TenTraicion) [inline index.html:1158]:
```suggestion
            <i class="fab fa-github"></i>
            <a href="https://github.com/kru2710shna" target="_blank">kru2710shna</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@kru2710shna, Thanks for your contribution. If you don't have twitter you can omit twitter from your card. also you need to add 2 more resources to make that a proper card. Kindly fix the social and add more resources and commit. Thanks

File: index.html [hunk 2]
```diff
@@ -1147,7 +1147,43 @@ <h2>A project for first-time contributions</h2>
...
+                  href="https://leetcode.com/krushna27doc/"
+                  target="_blank"
+                  title="Leetcode"
+                >
+                  Leetcode
+                </a>
+              </li>
+              <li>
+                <a
+                  href="krushna27doc@gmail.com"
```
Comment (TenTraicion) [inline index.html:1176]:
```suggestion
                  href="https://google.com"
```

File: index.html [hunk 3]
```diff
@@ -1147,7 +1147,43 @@ <h2>A project for first-time contributions</h2>
...
+                  target="_blank"
+                  title="Email"
```
Comment (TenTraicion) [inline index.html:1178]:
```suggestion
                  title="Google"
```

File: index.html [hunk 4]
```diff
@@ -1147,7 +1147,43 @@ <h2>A project for first-time contributions</h2>
...
+                >
+                  Contact Me
```
Comment (TenTraicion) [inline index.html:1180]:
```suggestion
                 Google
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@kru2710shna, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**